### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by avnerarbel1** [hossein-zare/react-native-dropdown-picker#757]
The DropDownPicker has several accessibility issues.
When in VoiceOver mode, the element should declare its role and state.

Level A - 1.3.1 Info and Relationships - 

An easy fix - add role and state to the dropdown:
`<TouchableOpacity accessibilityRole='combobox' accessibilityState={{expanded: open}} style={_style} onPress={__onPress} onLayout={__onLayout} {...props} ref={onRef} pointerEvents={pointerEvents} disabled={disabled} testID={testID}>`

In addition, the accessibility properties should be exported:
accessible
accessibilityRole
accessibilityState
accessibilityLabel
